<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniNotebook</class>
 <widget class="QDialog" name="MiniNotebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>177</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>177</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Мини блокнот</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/sZOInfo.png</normaloff>img/sZOInfo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
